pages/index.js

```diff
@@ -46,18 +48,25 @@ export default function HomePage() {
       <Header />
       <main>
         <h1>Add word</h1>
-        <Form onAddTranslations={handleAddTranslations} />
-        {isFound === "true" && <p>Wort bereits vorhanden</p>}
-        {isFound === "false" && <p>neues Wort hinzugefügt</p>}
 
-        {translationList.map((translation) => (
-          <ListEntry key={translation.id}>
-            <p>{translation.word}</p>
-            <small>({translation.language})</small>
-            <p>{translation.translated}</p>
-          </ListEntry>
-        ))}
+        <Form onAddTranslations={handleAddTranslations} />
+        <StyledSection>
+          {isFound === "true" && <p>word already exists</p>}
+          {isFound === "false" && (
```

Comment:
or

```js
isFound ? <>content when found</> : <>content when not found</>

Reply:
Thanks, I'll change this. 
One question: if I click the 'commit suggestion"- button below your code-example it will be implemented automatically?

Reply:
Tried to change it (both ways you suggested), but then it's not working anymore. The paragraphs are not disappearing anymore once they are displayed

Reply:
I wouldn't bet on that always working well. Rather push the changes from your local machine,

Reply:
Oh now I see it.. your "true" and "false" are strings.. that is why it is not the same to say

Try to not use "false" and "true" as strings, but as booleans instead.

Reply:
Okay, thanks. 
Concerning git: I added another commit on the same branch. will that be added automatically to the current PR? (because I don't see it here on this side)

Reply:
yes, if you commit to the same branch you already pushed and you push again, the commit will just be added on top

Reply:
Tried again. It's not working for the falsy value. The problem is to have an initial value that is either true nor false, so that the initial render will not display any of them

Reply:
Within your data-structure `isfound` needs to be a boolean, currently it seems to be a string.

Reply:
const [isFound, setIsFound] = useState(false);

and then you need to fill it with booleans instead of string:

setIsFound(false);
setIsFound(true);

currentiy you are doing it like this:
setIsFound(**"**true**"**);

Reply:
Changed it and it is working! 
But I don't understand why ;-) because now I set the initial state false, so I would expect that the text 'added new ...' will be displayed